static_src/actions/user_actions.js

```diff
@@ -119,6 +120,65 @@ const userActions = {
     });
   },
 
+  fetchUserInvite(email) {
+    AppDispatcher.handleViewAction({
+      type: userActionTypes.USER_INVITE_FETCH,
+      email
+    });
+
+    return uaaApi.getInviteUaaUser(email)
+      .then(data => userActions.receiveUserInvite(data));
+  },
+
+  receiveUserInvite(inviteData) {
+    AppDispatcher.handleServerAction({
+      type: userActionTypes.USER_INVITE_RECEIVED
+    });
+
+    const userGuid = inviteData.new_invites[0].userId;
+
+    return cfApi.postCreateNewUserWithGuid(userGuid)
+      .then(user => userActions.receiveUserInCF(user, inviteData));
+  },
+
+  receiveUserInCF(user, inviteData) {
+    AppDispatcher.handleServerAction({
+      type: userActionTypes.USER_IN_CF_CREATED,
+      user
+    });
+
+    if (user.guid) {
+      userActions.sendUserInviteEmail(inviteData);
+    }
+    // Once the user exists in CF, associate them to the organization.
+    return userActions.fetchUserAssociationToOrg(user);
+  },
+
+  sendUserInviteEmail(inviteData) {
+    AppDispatcher.handleServerAction({
+      type: userActionTypes.USER_EMAIL_INVITE
+    });
+    uaaApi.sendInviteEmail(inviteData);
+  },
+
+  fetchUserAssociationToOrg(user) {
+    AppDispatcher.handleServerAction({
+      type: userActionTypes.USER_ORG_ASSOCIATION_FETCH
+    });
+    const orgGuid = OrgStore.currentOrgGuid;
+
+    return cfApi.putAssociateUserToOrganization(user.guid, orgGuid)
+      .then(userActions.receivedUserAssociationToOrg(user, orgGuid));
+  },
+
+  receivedUserAssociationToOrg(user, orgGuid) {
```

Comment:
Like above, this action is after when the user was associated on the backend. So more like `associatedUserToOrg`.